ISMIR2021.tex

```diff
@@ -151,12 +155,86 @@ \section{Introduction}
             \item   Instead of using a single conference platform, we used a number of different platforms including Midspace, Gather.town, and Zoom to maximize the benefits of each platform. We focused on putting extra attention to ensure that we minimize the confusion that could arise due to platform switching. 
         \end{itemize}
 
-\ifthenelse{\equal{\showTodos}{true}}{
-\textcolor{red}{
 \section{Conference Platforms}
-    TODO: Not yet available.
-    }}{}
-    
+
+\textbf{Timeline}
+
+\begin{itemize}
+    \item 03/2021: Tech Chair committee established
+    \item 03/2021: Initial platform look-ups (Remo, Midspace, Gather)
+    \item 05/2021: Midspace negotiations started
+    \item late 08/2021: Contract with Midspace was signed (in the end through the ISMIR board)
+    \item Meantime: Getting scripts ready to include the schedule and content in Midspace
+    \item 08/2021: Gather.town booking request.
+    \item 10/2021: Gather.town map design started. We contracted a graphic designer to design the gather town map \href{https://www.fiverr.com/jc_shay/create-and-customize-your-gather-town-space-for-you}{via Fiverr}.
+    \item 10/10/2021: Mail with video information out to the authors (mirrored on ISMIR list)
+    \item 11/01/2021: Midspace opened its gates
+    \item 11/07 - 11/12/2021: Running ISMIR 2021
+\end{itemize}
+
+\subsection{Platform Overview}
+
+\begin{figure}[t]%
+    \centering
+    \includegraphics[width=0.9\textwidth]{fig/midspace_overview.pdf}%
+    \caption{Overview of Midspace's web-based graphical user interface. (a) Entry screen. (b) Conference schedule. (c) Detail page for a paper.}%
+    \label{fig:midspace}%
+\end{figure}
+
+Since ISMIR 2021 was fully digital, the conference platforms are the most essential part where all things glue together and most of the ``magic'' happens.
+
+\begin{itemize}
+    \item \textbf{Midspace:} Midspace is an all-in-one solution that provides a centralized web application to manage useful to schedule the conference and to assign respective roles to different sessions. For example, authors or session shares can be made fully responsible for their respective session content, sending reminders, let authors upload content etc. Figure~\ref{fig:midspace} shows excerpts from Midspace's graphical user interface. We decided to choose midspace as our centralized platform to orchestrate the conference. Midspace integrates a chat system, a video conferencing system (both async for keynotes and live for posters), a conference schedule/calendar and a system to let authors upload their content and videos themselves. This, midspaces replaced Slack (chat), miniconf (schedule website) and Dropbox (authors video upload). Furthermore, one of the main reasons where that we offered full support before and during the conference from the midspace team. 
```

Comment:
"that we offered full support": shouldn't this be that we WERE offered full support ?